airflow-core/src/airflow/ui/src/i18n/locales/de/components.json

```
@@ -0,0 +1,119 @@
+{
+  "backfill": {
+    "affectedMultiple": "{{count}} Läufe werden ausgelöst.",
+    "affectedNone": "Keine Läufe entsprechend en Kriterien.",
+    "affectedOne": "1 Lauf wird ausgelöst.",
+    "backwards": "Verarbeitung in Rückwärtiger Reihenfolge",
+    "dateRange": "Datumsbereich",
+    "dateRangeFrom": "Von",
+    "dateRangeTo": "Bis",
+    "errorStartDateBeforeEndDate": "Das Startdatum muss vor dem Enddatum liegen.",
+    "maxRuns": "Anzahl aktiver paralleler Läufe",
+    "reprocessBehavior": "Aufffüll-Modus",
+    "run": "Auffüllung starten",
+    "selectDescription": "Dieses Dag für einen Datumsbereich in der Vergangenheit ausführen",
```

```suggestion
    "selectDescription": "Diesen Dag für einen Datumsbereich in der Vergangenheit ausführen",
```

I so far assumed Dags are masculine, please ignore if you changed it to neuter :)